examples/table-resize/main.go

```diff
@@ -31,13 +31,12 @@ func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	return m, cmd
 }
 
-func (m model) View() string {
+func (m model) View(ctx tea.Context) string {
 	return "\n" + m.table.String() + "\n"
 }
 
 func main() {
-	re := lipgloss.NewRenderer(os.Stdout)
-	baseStyle := re.NewStyle().Padding(0, 1)
+	baseStyle := lipgloss.NewStyle().Padding(0, 1)
```

Comment:
The table initialization and creation should be inside `Init(ctx)` to be able to use `ctx.NewStyle()`